test/built-ins/ShadowRealm/prototype/evaluate/errors-from-the-other-realm-is-wrapped-into-a-typeerror.js

```diff
@@ -15,7 +15,7 @@ assert.sameValue(
 
 const r = new ShadowRealm();
 
-assert.throws(TypeError, () => r.evaluate('...'), 'SyntaxError => TypeError');
+assert.throws(SyntaxError, () => r.evaluate('...'), 'SyntaxError exposed to Parent');
```

Comment:
Yeah, this looks right to me, given step 5 here https://tc39.es/proposal-shadowrealm/#sec-performrealmeval.